content/develop/start/metamask.md

```diff
@@ -3,3 +3,155 @@ title: "Aurora: Getting Started with MetaMask"
 ---
 
 # Getting Started with MetaMask
+
+[MetaMask] is a convenient UI for interacting with Ethereum-compatible blockchains (such as Aurora).
+For the purpose of this guide, we will assume you are already familiar with MetaMask and have it installed.
+If you need help getting started with MetaMask itself, [check out their documentation](https://metamask.io/faqs.html).
+
+In this tutorial we will walk through connecting MetaMask to the Aurora TestNet, deploying a simple ERC-20 contract using [Remix], and transferring the new token using MetaMask.
+
+!!! note
+    Screenshots in this tutorial are taken from the MetaMask browser extension version 9.5.5.
+
+## Connecting MetaMask to Aurora
+
+In the top-right corner of the MetaMask interface, click the network selection drop-down and then click `Custom RPC`.
+
+![MetaMask-network-select](../../_img/metamask_choose_network.png)
+
+Fill in the form with the following information:
+
+* Network Name: Aurora TestNet
+* New RPC URL: `https://testnet.aurora.dev/`
+* Chain ID: 1313161555
+* Currency Symbol: ETH
+
+![MetaMask-create-aurora-rpc](../../_img/metamask_create_aurora_rpc.png)
```

Comment:
I think the only oversight in this tutorial was that the currency symbol should be set to ETH.